src/api/records.tsx

```diff
@@ -68,8 +68,29 @@ const fetchRecords = async (
 
   searchObj.push(...filtersObj);
 
-  if (searchObj.length > 0) {
-    queryParams.append('conditions', JSON.stringify({ $and: searchObj }));
+  const conditions: { [x: string]: { $exists: boolean } }[] = [];
```

Comment:
Can we rename this to something like `existsConditions` to be more explicit what it is?